packages/shared/chat/sagas.ts

```diff
@@ -285,22 +285,13 @@ function initChatCommands() {
       messageType: ChatMessageType.SYSTEM,
       sender: 'Decentraland',
       timestamp: Date.now(),
-      body: strings ? `Players around you:\n${strings}` : 'No other players are near to your location'
+      body: strings ? `Players around you:\n ${strings}` : 'No other players are near to your location'
     }
   })
 
-  addChatCommand('showfps', 'Show FPS counter', (_message) => {
-    fpsConfiguration.visible = !fpsConfiguration.visible
-    fpsConfiguration.visible ? getUnityInstance().ShowFPSPanel() : getUnityInstance().HideFPSPanel()
+  addChatCommand('debug', 'Show debug panel', (_message) => getDebugPanelMessage())
 
-    return {
-      messageId: uuid(),
-      sender: 'Decentraland',
-      messageType: ChatMessageType.SYSTEM,
-      timestamp: Date.now(),
-      body: 'Toggling FPS counter'
-    }
-  })
+  addChatCommand('showfps', 'Show fps panel', (_message) => getDebugPanelMessage())
```

Comment:
```suggestion
  addChatCommand('showfps', 'Show fps panel (deprecated in favor of /debug)', (_message) => getDebugPanelMessage())
```

Reply:
I don't feel strongly about such a long message, maybe we could hide it from the `/help` list in some other way?

Reply:
We have longer messages in the /help chat, currently it appears like this:

![Screenshot 2022-03-07 at 11 26 10](https://user-images.githubusercontent.com/4254522/157013687-d51716e5-3eb5-4f9c-9ccb-bf80d34a17a4.png)